dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/web/UpdateContainersPathsJob.java

```diff
@@ -0,0 +1,172 @@
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.exception.DotRuntimeException;
+import com.dotmarketing.exception.DotSecurityException;
+import com.dotmarketing.portlets.containers.business.ContainerExceptionNotifier;
+import com.dotmarketing.portlets.containers.business.ContainerFinderByIdOrPathStrategy;
+import com.dotmarketing.portlets.containers.business.FileAssetContainerUtil;
+import com.dotmarketing.portlets.containers.business.WorkingContainerFinderByIdOrPathStrategyResolver;
+import com.dotmarketing.portlets.containers.model.Container;
+import com.dotmarketing.portlets.contentlet.business.HostCache;
+import com.dotmarketing.portlets.templates.design.bean.TemplateLayout;
+import com.dotmarketing.quartz.DotStatefulJob;
+import com.dotmarketing.quartz.QuartzUtils;
+import com.dotmarketing.util.Logger;
+import org.jetbrains.annotations.NotNull;
+import org.quartz.*;
+
+import java.util.*;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+
+public class UpdateContainersPathsJob extends DotStatefulJob  {
+
+    final String GET_TEMPLATES_QUERY = "SELECT DISTINCT template.inode, template.drawed_body, template.body, identifier.host_inode " +
+            "FROM (identifier " +
+            "INNER JOIN template ON identifier.id = template.identifier) " +
+            "where template.drawed_body is not null and template.drawed_body LIKE ?";
+
+    @WrapInTransaction
+    @Override
+    public void run(final JobExecutionContext jobContext) throws JobExecutionException {
+        final JobDataMap jobDataMap = jobContext.getJobDetail().getJobDataMap();
+        final String oldHostName = jobDataMap.get("oldHostName").toString();
+        final String newHostName = jobDataMap.get("newHostName").toString();
+
+        try {
+            final List<Map<String, Object>> templates = getAllTemplatesByPath(oldHostName);
+
+            if (null != templates && !templates.isEmpty()) {
+                final Host host = APILocator.getHostAPI().find(templates.get(0).get("host_inode").toString(),
+                        APILocator.systemUser(), false);
+
+                final List<Params> params = getParameters(templates, oldHostName, newHostName);
+
+                if (!params.isEmpty()) {
+                    final DotConnect dotConnect = new DotConnect();
+                    dotConnect.executeBatch("update template set drawed_body = ?, body = ? where inode =?", params);
+                    cleanCache(templates, host);
+                }
+            }
+        } catch (DotDataException | DotSecurityException e) {
+            throw new JobExecutionException(e);
+        } finally {
+            try {
+                QuartzUtils.getSequentialScheduler().unscheduleJob(
+                        jobContext.getJobDetail().getName(),
+                        jobContext.getTrigger().getName()
+                );
+            } catch (SchedulerException e) {
+                e.printStackTrace();
```

Comment:
* Log exception and remove printstacktrace

Reply:
done https://github.com/dotCMS/core/pull/19059/commits/754b5ee095ddb38a0ee61855446b0f1f62085ed3#diff-f79af9d062935bd96345e8d64c411454R71